### PR TITLE
UX: Classes to investigate if should go to ADMINISTER

### PR DESCRIPTION
**Pull request by EstherAF**
[CTR-793](https://cloudbees.atlassian.net/browse/CTR-793)

### CODE REVIEW

**Comment by EstherAF** [comment]
The reason for every change can be read in every commit message

**Review by aHenryJard — APPROVED** [review]
lgtm

File: core/src/main/resources/hudson/PluginManager/sidepanel.groovy [hunk 1]
```diff
@@ -22,13 +22,16 @@
...
+import jenkins.model.Jenkins
+
 l=namespace(lib.LayoutTagLib)
 
 l.header()
 l.side_panel {
     l.tasks {
         l.task(icon:"icon-up icon-md", href:rootURL+'/', title:_("Back to Dashboard"))
-        l.task(icon:"icon-gear2 icon-md", href:"${rootURL}/manage", title:_("Manage Jenkins"), permission:app.CONFIGURE_JENKINS, it:app)
+        l.task(icon:"icon-gear2 icon-md", href:"${rootURL}/manage", title:_("Manage Jenkins"), permission:Jenkins.CONFIGURE_JENKINS, it:app)
```
Comment (aHenryJard) [inline core/src/main/resources/hudson/PluginManager/sidepanel.groovy:34]: why did you change app by Jenkins ?

File: core/src/main/resources/hudson/slaves/SlaveComputer/threadDump.jelly [hunk 2]
```diff
@@ -33,13 +33,13 @@ THE SOFTWARE.
     <st:include page="sidepanel.jelly" />
 
     <l:main-panel>
-      <l:isAdmin>
```
Comment (aHenryJard) [inline core/src/main/resources/hudson/slaves/SlaveComputer/threadDump.jelly:36]: good spot :+1:

**Review by aHenryJard — COMMENTED** [review]
Just one question, other than that it look good to me

**Review by aHenryJard — COMMENTED** [review]
I just have one question

File: core/src/main/resources/hudson/PluginManager/sidepanel.groovy (hunk 1 above)
Comment (EstherAF) [reply core/src/main/resources/hudson/PluginManager/sidepanel.groovy:34]: This was breaking, because app here is "Hudson" class, and that doesn't have the CONFIGURE_JENKINS property

Comment (aHenryJard) [reply core/src/main/resources/hudson/PluginManager/sidepanel.groovy:34]: ok.

**Review by aHenryJard — APPROVED** [review]
